cpp/scenario/gazebo/include/scenario/gazebo/utils.h

```diff
@@ -50,6 +50,7 @@ namespace scenario {
             std::string getModelFileFromFuel(const std::string& URI,
                                              const bool useCache = false);
             std::string getRandomString(const size_t length);
+            std::string getInstallPrefix();
```

Comment:
I suggest to document that this is only valid if `gym-ignition` is installed in `Developer` mode.

Reply:
Ok, I take advantage and I will document all the utils functions together in a new commit

Reply:
Done in https://github.com/robotology/gym-ignition/pull/176/commits/9e8f58e4ee3a06baad1e5d227f0a29d9f30b2a04